slither/tools/mutator/README.md

```diff
@@ -33,8 +33,7 @@ options:
   --timeout TIMEOUT     Set timeout for test command (by default 30 seconds)
   --output-dir OUTPUT_DIR
                         Name of output directory (by default 'mutation_campaign')
-  -v, --verbose         log mutants that are caught as well as those that are uncaught
-  -vv, --very-verbose   log mutants that are caught, uncaught, and fail to compile. And more!
```

Comment:
is this removal expected?

Reply:
Yes. It was done [here](https://github.com/crytic/slither/pull/2625/commits/9ea73293ad7de1a5a20a9526f55e7e258f25d47c) but the the CLI arg and README were not updated